### ISSUE TITLE
Confusing label for option about footers in "margin"

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#781]
On the Layout page is this option:

![image](https://user-images.githubusercontent.com/65363206/202852842-fa5835ef-ac14-4ae7-8532-087f06d0fdb8.png)

Again, maybe it is me, or maybe it is late, but this option seems to act exactly opposite to what the Help bubble says. Or at least after reading it several times and turning it on and off and watching the animation I still couldn't figure out what it would do for me (or not).

But by turning this off, I solved the weird behavior in #778. 

I don't know how it got turned on. I certainly never selected that check box.
Does anybody else find this option confusing?

**Comment by markpenny** [comment]
I have a feeling you're right! Nice catch! We used to have this option around the other way, but decided to swap the wording (and default state). BUT... promptly forgot about the (old) Tooltip (as shown here below). So I'll update it for the next build:
![image](https://user-images.githubusercontent.com/12962708/202909301-3a166392-62b1-4b6b-b0fd-76fb88fa580a.png)

**Comment by markpenny** [comment]
This is the new wording. Please feel free to suggest improvements to this:
![image](https://user-images.githubusercontent.com/12962708/202913163-60d08d01-a90b-407b-be90-e67bd28ce319.png)

**Comment by klassenjm** [comment]
Hi,

I saw David's comments here. I was also confused about the help text, and just tested the behaviour in order to understand how to set this.

In my view, checkboxes are easiest to understand quickly when the 'checked' state = the action being taken. Could this checkbox label say "Place footer text above the bottom margin"?

**Comment by markpenny** [comment]
Yes, I agree that wording is far more helpful. Thanks for the suggestion. I've updated it, and it will be in the next release.

**Comment by davidc86** [comment]
I like the new wording!
________________________________

**Comment by davidc86** [comment]
Oh, just one comment here, if this option is selected, then maybe those whose thinking requires this check box would prefer it if the wording of the other measurements would automatically change too? I don't think like they do, so maybe we need to ask them, but if selected, now the footer is INSIDE the text margin, so maybe the RED measurement is irrelevant, and the Blue measurement would become the main "bottom margin" measurement (because if that box is checked, then what they are saying is, "I want to control the size of the NON-PRINTABLE area on the page.") 

Yeah, this is pretty minor, but as is the UI might still be confusing to those who want to use this feature.

**Comment by davidc86** [comment]
I don't think this is an issue anymore. No one seems to find it weird. And I'm not sure what I was seeing and worried about. If this issue is still an issue, someone will create one. The main issue has been addressed nicely. Thanks.
